scripts/src/main/resources/scripts/command/ide

```diff
@@ -575,4 +575,6 @@ else
 fi
 
 echo
+# shellcheck disable=SC2007
+echo "$[devon_ide_version]" > "${DEVON_IDE_HOME}"/.devon.software.version
```

Comment:
This has to go to the end of the `doSetup()` function.
Otherwise it will always overwrite the version into that file whenever the `ide` commandlet is called.

Reply:
Of course, done